book/07-git-tools/sections/credentials.asc

```diff
@@ -83,15 +83,15 @@ username=bob
 password=s3cre7
 ----
 
-<1> This is the command line that initiates the interaction.
-<2> Git-credential is then waiting for input on stdin.
-    We provide it with the things we know: the protocol and hostname.
-<3> A blank line indicates that the input is complete, and the credential system should answer with what it knows.
-<4> Git-credential then takes over, and writes to stdout with the bits of information it found.
-<5> If credentials are not found, Git asks the user for the username and password, and provides them back to the invoking stdout (here they're attached to the same console).
+<1> Ito ay ang command line na nagpapasimula sa pakikipag-ugnayan.
+<2> Ang Git-credential ay naghihintay para sa input sa stdin.
+    Kami ay nagbibigay nito na may bagay na alam namin: ang protocol at hostname.
+<3> Isang blangko linya na nagpapahiwatig na ang input ay kumpleto, at ang credential na sistema ay dapat sagutin na kung ano ang alam.
```

Comment:
Isang blangko linya to Isang blangko na linya

Reply:
done change